scripts/tests/test_zkai_relabeling_benchmark_suite.py

```
@@ -91,12 +116,104 @@ def test_python_reference_suite_rejects_all_mutations(self) -> None:
+        harness_path = SUITE.HARNESS_PATH.resolve()
+        forbidden_module_names = {
+            "agent_step_receipt_relabeling_harness",
+            "agent_step_receipt_harness",
+            "scripts.agent_step_receipt_relabeling_harness",
+        }
+        before_harness_modules = self._module_names_loaded_from(harness_path)
+        before_names = set(sys.modules)
+        original_spec_from_file_location = importlib.util.spec_from_file_location
+        original_import = builtins.__import__
+
+        def guarded_spec_from_file_location(name, location, *args, **kwargs):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `guarded_spec_from_file_location`

(ANN202)